packages/bpk-component-modal/src/bpk-modal.scss

```diff
@@ -18,6 +18,12 @@
 
 @import '~bpk-mixins/index';
 
+@mixin fullscreen() {
+  display: block;
```

Comment:
I tested this across a few mobile and desktop browsers. One thing I noticed on desktop is that clicking on the area below the modal dialog content closes the modal. Once solution for this is to keep the container as `display: flex;` which makes any block-level children occupy the full height and width of the container.

Reply:
Wau good catch. I used `block` because before was for mobile fullscreen `block` as well. But yes it is possible to close it in the way you mentioned. Updated for `Flex` and it looks ok as well.